R/install-remote.R

```diff
@@ -17,12 +17,12 @@ install_remote <- function(remote, ..., quiet = FALSE) {
   source <- source_pkg(bundle, subdir = remote$subdir)
   on.exit(unlink(source, recursive = TRUE), add = TRUE)
 
-  add_metadata(source, remote_metadata(remote, bundle, source))
+  metadata <- remote_metadata(remote, bundle, source)
 
   # Because we've modified DESCRIPTION, its original MD5 value is wrong
   clear_description_md5(source)
```

Comment:
removed call to `clear_description_md5` in 6467f7c, running tests suggests it doesn't break anything. I wonder if the function is needed at all in the code anymore? I think this was the only place it was being called.